docs/website/versioned_docs/version-3.0.0/command-reference/dev.md

```diff
@@ -0,0 +1,226 @@
+---
+title: odo dev
+sidebar_position: 2
+---
+
+## Description
+
+`odo dev` is used in order to quickly and effectively iterate through your development process for building an application.
+
+This is [inner loop](http://localhost:3000/docs/3.0.0/intro#what-is-inner-loop-and-outer-loop) development and allows you to code, build, run and test the application in a continuous workflow.
+
+## Running the Command
+
+If you haven't already done so, you must [initialize](../command-reference/init) your source code with the `odo init` command.
+
+Afterwards, run `odo dev`:
+
+```sh
+$ odo dev
+  __
+ /  \__     Developing using the my-nodejs-app Devfile
+ \__/  \    Namespace: default
+ /  \__/    odo version: v3.0.0-alpha1
+ \__/
+
+↪ Deploying to the cluster in developer mode
+ ✓  Waiting for Kubernetes resources [3s]
+ ✓  Syncing files into the container [335ms]
+ ✓  Building your application in container on cluster [2s]
+ ✓  Executing the application [1s]
+
+Your application is now running on the cluster
+ - Forwarding from 127.0.0.1:40001 -> 3000
+
+Watching for changes in the current directory /Users/user/nodejs
+Press Ctrl+c to exit `odo dev` and delete resources from the cluster
+```
+
+In the above example, three things have happened:
+  * Your application has been built and deployed to the cluster
+  * `odo` has port-forwarded your application for local accessability
+  * `odo` will watch for changes in the current directory and rebuild the application when changes are detected
+
+## Devfile (Advanced Usage)
+
+
+### Description
+
+When `odo dev` is ran, it first looks in the `devfile.yaml` for the instructions to be executed, specifically: `components` and `commands`.
```

Comment:
I'm not a native English speaker, but it should be "is run", no? ^^

```suggestion
When `odo dev` is run, it first looks in the `devfile.yaml` for the instructions to be executed, specifically: `components` and `commands`.
```

Reply:
Nevermind! It's suppose to be ran according to google.